<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>92</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>92</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Manual x Axis</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="auto_rescale">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>AUTO RESCALE</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>52</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>OR</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>53</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>X min</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="x_min_value"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>52</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X max</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="x_max_value"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>x_max_value</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>x_max_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_min_value</sender>
   <signal>returnPressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>x_min_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_rescale</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>x_auto_rescale_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>x_max_event()</slot>
  <slot>x_min_event()</slot>
  <slot>x_auto_rescale_event()</slot>
 </slots>
</ui>
